functions/playground/proxy.js

```diff
@@ -0,0 +1,68 @@
+// json-ld.org playground proxy
+//
+// details:
+// - Basic proxy for use with the json-ld.org playground.
+// - Built for deployment using Cloudflare Pages Functions API
+//   - https://developers.cloudflare.com/pages/functions/
+// - Only handle GET requests.
+// - Only designed to be used by the playground, no CORS support.
+// - Only handle requests for 'http:' URLs.
```

Comment:
Can you explain the `http:` only thing? I'm confused by how this'll work in practice since context URLs are often served from `https:` URLs (and should be). I'm likely missing something...

Reply:
The purpose of the proxy is to avoid browser security errors and allow the https site to load http resources.  If you look at playground.js you can see that's the only time it's used.  Best practice these days is to use https, but that's not required, and historically it was enough of an issue that we wrote the php proxy.